README.md

```diff
@@ -26,6 +26,11 @@ npm install -D @types/vscode
 }
 ```
 
+If you use Vite, add in your vite.config.js:
+```js
+assetsInclude: ['**/*.wasm']
```

Comment:
It works well in the demo here because there is nothing else, but it may break application already using wasm via the wasm loader. Maybe the glob should be more specific, don't you think?

Reply:
I changed it to `node_modules/vscode-oniguruma/release/onig.wasm`. This works right now but could break with library updates if the path were to change.

Reply:
Maybe you can includes all wasm inside vscode-oniguruma then?

Reply:
I don't know why I didn't think of that. Great idea!